### PR TITLE
[11.0] Add `website_addthis`

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: website_addthis/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+1. go to Website -> Settings
```
Comment (tarteo) [inline website_addthis/readme/CONFIGURE.rst]: You can use #. here.

**Review by tarteo — CHANGES_REQUESTED** [review]

File: website_addthis/readme/CONFIGURE.rst (hunk 1 above)
Comment (simahawk) [reply website_addthis/readme/CONFIGURE.rst]: AFAIK it's supported in markdown only

File: website_addthis/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "AddThis integration",
+    "version": "11.0.1.0.0",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Website",
+    "summary": "Integrate addthis.com features in your website",
```
Comment (MeetKD) [inline website_addthis/__manifest__.py:10]: Website key is missing !

**Review by MeetKD — CHANGES_REQUESTED** [review]
Nice module by d way !! :+1:

File: website_addthis/__manifest__.py (hunk 2 above)
Comment (simahawk) [reply website_addthis/__manifest__.py:10]: fixed, tnx :)

Comment (MeetKD) [reply website_addthis/__manifest__.py:10]: @simahawk :+1:

**Review by grindtildeath — APPROVED** [review]

**Comment by simahawk** [comment]
@MeetKD @tarteo can you update your reviews?

File: website_addthis/readme/CONFIGURE.rst (hunk 1 above)
Comment (simahawk) [reply website_addthis/readme/CONFIGURE.rst]: TIL I learned that `#` is for mkdown and `#.` for RST :smile: fixed

**Review by tarteo — APPROVED** [review]
@simahawk Thanks for the changes. :+1:

**Comment by simahawk** [comment]
@MeetKD change addressed, I assume he's going to give green light :stuck_out_tongue:

**Pull request merged**